<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<ns2:test-suite xmlns:ns2="urn:model.allure.qatools.yandex.ru" start="1576912714857" stop="1576912736135" version="1.4.8">
    <name>Default Suite</name>
    <title>Default Suite : seleniumCp[]</title>
    <test-cases>
        <test-case start="1576912735762" stop="1576912735770" status="canceled">
            <name>loginBefore</name>
            <description></description>
            <failure>
                <message>WebDriverException: Returned value cannot be converted to WebElement: {stacktrace=Backtrace:
	Ordinal0 [0x0115A903+1550595]
	Ordinal0 [0x010DA701+1025793]
	Ordinal0 [0x0105C58E+509326]
	Ordinal0 [0x00FF0FB9+69561]
	Ordinal0 [0x01008DD7+167383]
	Ordinal0 [0x010077F2+161778]
	Ordinal0 [0x010073CA+160714]
	Ordinal0 [0x00FE221E+8734]
	Ordinal0 [0x00FE2606+9734]
	Ordinal0 [0x00FE2C80+11392]
	Ordinal0 [0x010F42C7+1131207]
	GetHandleVerifier [0x011F70FD+523789]
	GetHandleVerifier [0x011F6E90+523168]
	GetHandleVerifier [0x011FE1E7+552695]
	GetHandleVerifier [0x011F78FA+525834]
	Ordinal0 [0x010EB7FC+1095676]
	Ordinal0 [0x00FE2053+8275]
	Ordinal0 [0x00FE1DA7+7591]
	GetHandleVerifier [0x0155A3BC+4075724]
	BaseThreadInitThunk [0x75BD6359+25]
	RtlGetAppContainerNamedObjectPath [0x77B07B74+228]
	RtlGetAppContainerNamedObjectPath [0x77B07B44+180]
, error=invalid session id, message=invalid session id}
Build info: version: '3.0.0', revision: '350cf60', time: '2016-10-13 10:48:57 -0700'
System info: host: 'DESKTOP-8DI1090', ip: '10.1.101.3', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_231'
Driver info: driver.version: RemoteWebDriver</message>
                <stack-trace>org.openqa.selenium.WebDriverException: Returned value cannot be converted to WebElement: {stacktrace=Backtrace:
	Ordinal0 [0x0115A903+1550595]
	Ordinal0 [0x010DA701+1025793]
	Ordinal0 [0x0105C58E+509326]
	Ordinal0 [0x00FF0FB9+69561]
	Ordinal0 [0x01008DD7+167383]
	Ordinal0 [0x010077F2+161778]
	Ordinal0 [0x010073CA+160714]
	Ordinal0 [0x00FE221E+8734]
	Ordinal0 [0x00FE2606+9734]
	Ordinal0 [0x00FE2C80+11392]
	Ordinal0 [0x010F42C7+1131207]
	GetHandleVerifier [0x011F70FD+523789]
	GetHandleVerifier [0x011F6E90+523168]
	GetHandleVerifier [0x011FE1E7+552695]
	GetHandleVerifier [0x011F78FA+525834]
	Ordinal0 [0x010EB7FC+1095676]
	Ordinal0 [0x00FE2053+8275]
	Ordinal0 [0x00FE1DA7+7591]
	GetHandleVerifier [0x0155A3BC+4075724]
	BaseThreadInitThunk [0x75BD6359+25]
	RtlGetAppContainerNamedObjectPath [0x77B07B74+228]
	RtlGetAppContainerNamedObjectPath [0x77B07B44+180]
, error=invalid session id, message=invalid session id}
Build info: version: '3.0.0', revision: '350cf60', time: '2016-10-13 10:48:57 -0700'
System info: host: 'DESKTOP-8DI1090', ip: '10.1.101.3', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_231'
Driver info: driver.version: RemoteWebDriver
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:375)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:473)
	at org.openqa.selenium.By$ByXPath.findElement(By.java:361)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:360)
	at SeleniumMethod.WebDriverUtil.adminLogin(WebDriverUtil.java:76)
	at complex.ReportManagement.loginBefore(ReportManagement.java:23)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:510)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:211)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.TestRunner.beforeRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:616)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
Caused by: java.lang.ClassCastException: com.google.common.collect.Maps$TransformedEntriesMap cannot be cast to org.openqa.selenium.WebElement
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:373)
	... 26 more
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="DESKTOP-8DI1090"/>
                <label name="thread" value="15520@DESKTOP-8DI1090.main"/>
            </labels>
        </test-case>
        <test-case start="1576912735852" stop="1576912735979" status="canceled">
            <name>vipNumberToday</name>
            <title>验证总代理报表本月会员数</title>
            <description type="text">测试仪表盘总会员数是否等于会员列表的会员数</description>
            <failure>
                <message>SkipException: The test was skipped for some reason</message>
                <stack-trace>org.testng.SkipException: The test was skipped for some reason
	at ru.yandex.qatools.allure.testng.AllureTestListener.onTestSkipped(AllureTestListener.java:148)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1678)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1670)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="severity" value="blocker"/>
                <label name="issue" value="http://10.1.101.66:890/index.php?m=testcase&amp;f=view&amp;caseID=25&amp;version=1"/>
                <label name="story" value="总代理报表"/>
                <label name="feature" value="报表管理"/>
                <label name="host" value="DESKTOP-8DI1090"/>
                <label name="thread" value="15520@DESKTOP-8DI1090.main"/>
            </labels>
        </test-case>
        <test-case start="1576912735981" stop="1576912735984" status="canceled">
            <name>countTotal</name>
            <title>验证总代理报表本月充值金额</title>
            <description type="text">测试总代理报表本月充值笔数是否等于跳转后的代理报表的充值笔数总和</description>
            <failure>
                <message>SkipException: The test was skipped for some reason</message>
                <stack-trace>org.testng.SkipException: The test was skipped for some reason
	at ru.yandex.qatools.allure.testng.AllureTestListener.onTestSkipped(AllureTestListener.java:148)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1678)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1670)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="severity" value="blocker"/>
                <label name="issue" value="http://10.1.101.66:890/index.php?m=testcase&amp;f=view&amp;caseID=26&amp;version=1"/>
                <label name="story" value="总代理报表"/>
                <label name="feature" value="报表管理"/>
                <label name="host" value="DESKTOP-8DI1090"/>
                <label name="thread" value="15520@DESKTOP-8DI1090.main"/>
            </labels>
        </test-case>
        <test-case start="1576912735986" stop="1576912735988" status="canceled">
            <name>betMoneyAgentTotal</name>
            <title>验证总代理报表本月投注金额</title>
            <description type="text">判断总代理报表的投注金额是否等于跳转后的代理报表投注金额总和</description>
            <failure>
                <message>SkipException: The test was skipped for some reason</message>
                <stack-trace>org.testng.SkipException: The test was skipped for some reason
	at ru.yandex.qatools.allure.testng.AllureTestListener.onTestSkipped(AllureTestListener.java:148)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1678)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1670)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="severity" value="blocker"/>
                <label name="issue" value="http://10.1.101.66:890/index.php?m=testcase&amp;f=view&amp;caseID=27&amp;version=1"/>
                <label name="story" value="总代理报表"/>
                <label name="feature" value="报表管理"/>
                <label name="host" value="DESKTOP-8DI1090"/>
                <label name="thread" value="15520@DESKTOP-8DI1090.main"/>
            </labels>
        </test-case>
        <test-case start="1576912735990" stop="1576912735993" status="canceled">
            <name>winBetMoneyAgentTotal</name>
            <title>验证总代理报表本月赢利投注金额</title>
            <description type="text">判断总代理报表的赢利投注金额是否等于跳转后的代理报表赢利投注金额总和</description>
            <failure>
                <message>SkipException: The test was skipped for some reason</message>
                <stack-trace>org.testng.SkipException: The test was skipped for some reason
	at ru.yandex.qatools.allure.testng.AllureTestListener.onTestSkipped(AllureTestListener.java:148)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1678)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1670)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="severity" value="blocker"/>
                <label name="issue" value="http://10.1.101.66:890/index.php?m=testcase&amp;f=view&amp;caseID=28&amp;version=1"/>
                <label name="story" value="总代理报表"/>
                <label name="feature" value="报表管理"/>
                <label name="host" value="DESKTOP-8DI1090"/>
                <label name="thread" value="15520@DESKTOP-8DI1090.main"/>
            </labels>
        </test-case>
        <test-case start="1576912735996" stop="1576912735998" status="canceled">
            <name>serviceChargeAgentTotal</name>
            <title>验证总代理报表本月充值优惠/服务费金额</title>
            <description type="text">判断总代理报表的充值优惠/手续费金额是否等于跳转后的代理报表充值优惠/手续费金额总和</description>
            <failure>
                <message>SkipException: The test was skipped for some reason</message>
                <stack-trace>org.testng.SkipException: The test was skipped for some reason
	at ru.yandex.qatools.allure.testng.AllureTestListener.onTestSkipped(AllureTestListener.java:148)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1678)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1670)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="severity" value="blocker"/>
                <label name="issue" value="http://10.1.101.66:890/index.php?m=testcase&amp;f=view&amp;caseID=29&amp;version=1"/>
                <label name="story" value="总代理报表"/>
                <label name="feature" value="报表管理"/>
                <label name="host" value="DESKTOP-8DI1090"/>
                <label name="thread" value="15520@DESKTOP-8DI1090.main"/>
            </labels>
        </test-case>
        <test-case start="1576912736001" stop="1576912736004" status="canceled">
            <name>waterBreakAgentTotal</name>
            <title>验证总代理报表本月代理退水金额</title>
            <description type="text">判断总代理报表的代理退水金额是否等于跳转后的代理报表代理退水金额总和</description>
            <failure>
                <message>SkipException: The test was skipped for some reason</message>
                <stack-trace>org.testng.SkipException: The test was skipped for some reason
	at ru.yandex.qatools.allure.testng.AllureTestListener.onTestSkipped(AllureTestListener.java:148)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1678)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1670)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="severity" value="blocker"/>
                <label name="issue" value="http://10.1.101.66:890/index.php?m=testcase&amp;f=view&amp;caseID=30&amp;version=1"/>
                <label name="story" value="总代理报表"/>
                <label name="feature" value="报表管理"/>
                <label name="host" value="DESKTOP-8DI1090"/>
                <label name="thread" value="15520@DESKTOP-8DI1090.main"/>
            </labels>
        </test-case>
        <test-case start="1576912736006" stop="1576912736009" status="canceled">
            <name>oddsMoneyAgentTotal</name>
            <title>验证总代理报表本月代理赔率金额</title>
            <description type="text">判断总代理报表的代理赔率金额是否等于跳转后的代理报表代理赔率金额总和</description>
            <failure>
                <message>SkipException: The test was skipped for some reason</message>
                <stack-trace>org.testng.SkipException: The test was skipped for some reason
	at ru.yandex.qatools.allure.testng.AllureTestListener.onTestSkipped(AllureTestListener.java:148)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1678)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1670)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="severity" value="blocker"/>
                <label name="issue" value="http://10.1.101.66:890/index.php?m=testcase&amp;f=view&amp;caseID=31&amp;version=1"/>
                <label name="story" value="总代理报表"/>
                <label name="feature" value="报表管理"/>
                <label name="host" value="DESKTOP-8DI1090"/>
                <label name="thread" value="15520@DESKTOP-8DI1090.main"/>
            </labels>
        </test-case>
        <test-case start="1576912736011" stop="1576912736012" status="canceled">
            <name>loseWinAgentTotal</name>
            <title>验证总代理报表本月会员输赢(不包括退水)金额</title>
            <description type="text">判断总代理报表的会员输赢（不包括退水）金额是否等于跳转后的代理报表会员输赢（不包括退水）金额总和</description>
            <failure>
                <message>SkipException: The test was skipped for some reason</message>
                <stack-trace>org.testng.SkipException: The test was skipped for some reason
	at ru.yandex.qatools.allure.testng.AllureTestListener.onTestSkipped(AllureTestListener.java:148)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1678)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1670)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="severity" value="blocker"/>
                <label name="issue" value="http://10.1.101.66:890/index.php?m=testcase&amp;f=view&amp;caseID=32&amp;version=1"/>
                <label name="story" value="总代理报表"/>
                <label name="feature" value="报表管理"/>
                <label name="host" value="DESKTOP-8DI1090"/>
                <label name="thread" value="15520@DESKTOP-8DI1090.main"/>
            </labels>
        </test-case>
        <test-case start="1576912736014" stop="1576912736019" status="canceled">
            <name>waterBreakPracticeAgentTotal</name>
            <title>验证总代理报表本月实际退水金额</title>
            <description type="text">判断总代理报表的实际退水金额是否等于跳转后的代理报表实际退水金额总和</description>
            <failure>
                <message>SkipException: The test was skipped for some reason</message>
                <stack-trace>org.testng.SkipException: The test was skipped for some reason
	at ru.yandex.qatools.allure.testng.AllureTestListener.onTestSkipped(AllureTestListener.java:148)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1678)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1670)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="severity" value="blocker"/>
                <label name="issue" value="http://10.1.101.66:890/index.php?m=testcase&amp;f=view&amp;caseID=33&amp;version=1"/>
                <label name="story" value="总代理报表"/>
                <label name="feature" value="报表管理"/>
                <label name="host" value="DESKTOP-8DI1090"/>
                <label name="thread" value="15520@DESKTOP-8DI1090.main"/>
            </labels>
        </test-case>
        <test-case start="1576912736021" stop="1576912736023" status="canceled">
            <name>loseWinAndWaterAgentTotal</name>
            <title>验证总代理报表本月实际输赢(包括退水)金额</title>
            <description type="text">判断总代理报表的实际输赢(包括退水)金额是否等于跳转后的代理报表实际输赢(包括退水)金额总和</description>
            <failure>
                <message>SkipException: The test was skipped for some reason</message>
                <stack-trace>org.testng.SkipException: The test was skipped for some reason
	at ru.yandex.qatools.allure.testng.AllureTestListener.onTestSkipped(AllureTestListener.java:148)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1678)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1670)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="severity" value="blocker"/>
                <label name="issue" value="http://10.1.101.66:890/index.php?m=testcase&amp;f=view&amp;caseID=34&amp;version=1"/>
                <label name="story" value="总代理报表"/>
                <label name="feature" value="报表管理"/>
                <label name="host" value="DESKTOP-8DI1090"/>
                <label name="thread" value="15520@DESKTOP-8DI1090.main"/>
            </labels>
        </test-case>
        <test-case start="1576912736024" stop="1576912736027" status="canceled">
            <name>vipNumberAgent</name>
            <title>验证代理报表本月会员数量</title>
            <description type="text">判断代理报表的本月会员数是否等于跳转后的代理报表本月会员数总和</description>
            <failure>
                <message>SkipException: The test was skipped for some reason</message>
                <stack-trace>org.testng.SkipException: The test was skipped for some reason
	at ru.yandex.qatools.allure.testng.AllureTestListener.onTestSkipped(AllureTestListener.java:148)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1678)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1670)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="severity" value="blocker"/>
                <label name="issue" value="http://10.1.101.66:890/index.php?m=testcase&amp;f=view&amp;caseID=35&amp;version=1"/>
                <label name="story" value="代理报表"/>
                <label name="feature" value="报表管理"/>
                <label name="host" value="DESKTOP-8DI1090"/>
                <label name="thread" value="15520@DESKTOP-8DI1090.main"/>
            </labels>
        </test-case>
        <test-case start="1576912736029" stop="1576912736031" status="canceled">
            <name>rechargeMoneyAgent</name>
            <title>验证代理报表本月充值金额</title>
            <description type="text">判断代理报表的本月充值金额是否等于跳转后的代理报表本月充值金额总和</description>
            <failure>
                <message>SkipException: The test was skipped for some reason</message>
                <stack-trace>org.testng.SkipException: The test was skipped for some reason
	at ru.yandex.qatools.allure.testng.AllureTestListener.onTestSkipped(AllureTestListener.java:148)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1678)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1670)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="severity" value="blocker"/>
                <label name="issue" value="http://10.1.101.66:890/index.php?m=testcase&amp;f=view&amp;caseID=36&amp;version=1"/>
                <label name="story" value="代理报表"/>
                <label name="feature" value="报表管理"/>
                <label name="host" value="DESKTOP-8DI1090"/>
                <label name="thread" value="15520@DESKTOP-8DI1090.main"/>
            </labels>
        </test-case>
        <test-case start="1576912736035" stop="1576912736038" status="canceled">
            <name>countAgent</name>
            <title>验证代理报表本月充值笔数</title>
            <description type="text">判断代理报表的本月笔数是否等于跳转后的代理报表本月笔数总和</description>
            <failure>
                <message>SkipException: The test was skipped for some reason</message>
                <stack-trace>org.testng.SkipException: The test was skipped for some reason
	at ru.yandex.qatools.allure.testng.AllureTestListener.onTestSkipped(AllureTestListener.java:148)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1678)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1670)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="severity" value="blocker"/>
                <label name="issue" value="http://10.1.101.66:890/index.php?m=testcase&amp;f=view&amp;caseID=37&amp;version=1"/>
                <label name="story" value="代理报表"/>
                <label name="feature" value="报表管理"/>
                <label name="host" value="DESKTOP-8DI1090"/>
                <label name="thread" value="15520@DESKTOP-8DI1090.main"/>
            </labels>
        </test-case>
        <test-case start="1576912736041" stop="1576912736044" status="canceled">
            <name>betMoneyAgentReport</name>
            <title>验证代理报表本月投注金额</title>
            <description type="text">判断代理报表的本月投注金额是否等于跳转后的代理报表本月投注金额总和</description>
            <failure>
                <message>SkipException: The test was skipped for some reason</message>
                <stack-trace>org.testng.SkipException: The test was skipped for some reason
	at ru.yandex.qatools.allure.testng.AllureTestListener.onTestSkipped(AllureTestListener.java:148)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1678)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1670)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="severity" value="blocker"/>
                <label name="issue" value="http://10.1.101.66:890/index.php?m=testcase&amp;f=view&amp;caseID=38&amp;version=1"/>
                <label name="story" value="代理报表"/>
                <label name="feature" value="报表管理"/>
                <label name="host" value="DESKTOP-8DI1090"/>
                <label name="thread" value="15520@DESKTOP-8DI1090.main"/>
            </labels>
        </test-case>
        <test-case start="1576912736046" stop="1576912736049" status="canceled">
            <name>winBetMoneyAgentReport</name>
            <title>验证代理报表本月赢利投注金额</title>
            <description type="text">判断代理报表的本月赢利投注金额是否等于跳转后的代理报表本月赢利投注金额总和</description>
            <failure>
                <message>SkipException: The test was skipped for some reason</message>
                <stack-trace>org.testng.SkipException: The test was skipped for some reason
	at ru.yandex.qatools.allure.testng.AllureTestListener.onTestSkipped(AllureTestListener.java:148)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1678)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1670)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="severity" value="blocker"/>
                <label name="issue" value="http://10.1.101.66:890/index.php?m=testcase&amp;f=view&amp;caseID=39&amp;version=1"/>
                <label name="story" value="代理报表"/>
                <label name="feature" value="报表管理"/>
                <label name="host" value="DESKTOP-8DI1090"/>
                <label name="thread" value="15520@DESKTOP-8DI1090.main"/>
            </labels>
        </test-case>
        <test-case start="1576912736051" stop="1576912736054" status="canceled">
            <name>serviceMoneyAgent</name>
            <title>验证代理报表本月充值优惠/手续费</title>
            <description type="text">判断代理报表的本月充值优惠/手续费金额是否等于跳转后的代理报表本月充值优惠/手续费金额总和</description>
            <failure>
                <message>SkipException: The test was skipped for some reason</message>
                <stack-trace>org.testng.SkipException: The test was skipped for some reason
	at ru.yandex.qatools.allure.testng.AllureTestListener.onTestSkipped(AllureTestListener.java:148)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1678)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1670)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="severity" value="blocker"/>
                <label name="issue" value="http://10.1.101.66:890/index.php?m=testcase&amp;f=view&amp;caseID=40&amp;version=1"/>
                <label name="story" value="代理报表"/>
                <label name="feature" value="报表管理"/>
                <label name="host" value="DESKTOP-8DI1090"/>
                <label name="thread" value="15520@DESKTOP-8DI1090.main"/>
            </labels>
        </test-case>
        <test-case start="1576912736055" stop="1576912736058" status="canceled">
            <name>oddsAgent</name>
            <title>验证代理报表本月代理赔率金额</title>
            <description type="text">判断代理报表的本月代理赔率金额是否等于跳转后的代理报表本月代理赔率金额总和</description>
            <failure>
                <message>SkipException: The test was skipped for some reason</message>
                <stack-trace>org.testng.SkipException: The test was skipped for some reason
	at ru.yandex.qatools.allure.testng.AllureTestListener.onTestSkipped(AllureTestListener.java:148)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1678)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1670)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="severity" value="blocker"/>
                <label name="issue" value="http://10.1.101.66:890/index.php?m=testcase&amp;f=view&amp;caseID=41&amp;version=1"/>
                <label name="story" value="代理报表"/>
                <label name="feature" value="报表管理"/>
                <label name="host" value="DESKTOP-8DI1090"/>
                <label name="thread" value="15520@DESKTOP-8DI1090.main"/>
            </labels>
        </test-case>
        <test-case start="1576912736060" stop="1576912736063" status="canceled">
            <name>waterBreakMoneyAgent</name>
            <title>验证代理报表本月代理退水金额</title>
            <description type="text">判断代理报表的本月代理退水金额是否等于跳转后的代理报表本月代理退水金额总和</description>
            <failure>
                <message>SkipException: The test was skipped for some reason</message>
                <stack-trace>org.testng.SkipException: The test was skipped for some reason
	at ru.yandex.qatools.allure.testng.AllureTestListener.onTestSkipped(AllureTestListener.java:148)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1678)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1670)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="severity" value="blocker"/>
                <label name="issue" value="http://10.1.101.66:890/index.php?m=testcase&amp;f=view&amp;caseID=42&amp;version=1"/>
                <label name="story" value="代理报表"/>
                <label name="feature" value="报表管理"/>
                <label name="host" value="DESKTOP-8DI1090"/>
                <label name="thread" value="15520@DESKTOP-8DI1090.main"/>
            </labels>
        </test-case>
        <test-case start="1576912736066" stop="1576912736068" status="canceled">
            <name>winLoseNotWaterAgent</name>
            <title>验证代理报表本月实际输赢（不包括退水）</title>
            <description type="text">判断代理报表的本月会员输赢（不包括退水）是否等于跳转后的代理报表本月会员输赢（不包括退水）总和</description>
            <failure>
                <message>SkipException: The test was skipped for some reason</message>
                <stack-trace>org.testng.SkipException: The test was skipped for some reason
	at ru.yandex.qatools.allure.testng.AllureTestListener.onTestSkipped(AllureTestListener.java:148)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1678)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1670)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="severity" value="blocker"/>
                <label name="issue" value="http://10.1.101.66:890/index.php?m=testcase&amp;f=view&amp;caseID=43&amp;version=1"/>
                <label name="story" value="代理报表"/>
                <label name="feature" value="报表管理"/>
                <label name="host" value="DESKTOP-8DI1090"/>
                <label name="thread" value="15520@DESKTOP-8DI1090.main"/>
            </labels>
        </test-case>
        <test-case start="1576912736070" stop="1576912736072" status="canceled">
            <name>winLoseWaterAgent</name>
            <title>验证代理报表本月实际输赢（包括退水）</title>
            <description type="text">判断代理报表的本月会员输赢（包括退水）是否等于跳转后的代理报表本月会员输赢（包括退水）总和</description>
            <failure>
                <message>SkipException: The test was skipped for some reason</message>
                <stack-trace>org.testng.SkipException: The test was skipped for some reason
	at ru.yandex.qatools.allure.testng.AllureTestListener.onTestSkipped(AllureTestListener.java:148)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1678)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1670)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="severity" value="blocker"/>
                <label name="issue" value="http://10.1.101.66:890/index.php?m=testcase&amp;f=view&amp;caseID=44&amp;version=1"/>
                <label name="story" value="代理报表"/>
                <label name="feature" value="报表管理"/>
                <label name="host" value="DESKTOP-8DI1090"/>
                <label name="thread" value="15520@DESKTOP-8DI1090.main"/>
            </labels>
        </test-case>
        <test-case start="1576912736074" stop="1576912736076" status="canceled">
            <name>waterBreakMoneyActivityAgent</name>
            <title>验证代理报表本月实际退水金额</title>
            <description type="text">判断代理报表的本月代理实际退水是否等于跳转后的代理报表本月代理实际退水总和</description>
            <failure>
                <message>SkipException: The test was skipped for some reason</message>
                <stack-trace>org.testng.SkipException: The test was skipped for some reason
	at ru.yandex.qatools.allure.testng.AllureTestListener.onTestSkipped(AllureTestListener.java:148)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1678)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1670)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="severity" value="blocker"/>
                <label name="issue" value="http://10.1.101.66:890/index.php?m=testcase&amp;f=view&amp;caseID=45&amp;version=1"/>
                <label name="story" value="代理报表"/>
                <label name="feature" value="报表管理"/>
                <label name="host" value="DESKTOP-8DI1090"/>
                <label name="thread" value="15520@DESKTOP-8DI1090.main"/>
            </labels>
        </test-case>
        <test-case start="1576912736078" stop="1576912736081" status="canceled">
            <name>rechargeMoneyVip</name>
            <title>验证会员报表本月充值金额</title>
            <description type="text">判断会员报表的充值金额是否等于该会员充值记录的充值总计</description>
            <failure>
                <message>SkipException: The test was skipped for some reason</message>
                <stack-trace>org.testng.SkipException: The test was skipped for some reason
	at ru.yandex.qatools.allure.testng.AllureTestListener.onTestSkipped(AllureTestListener.java:148)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1678)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1670)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="severity" value="blocker"/>
                <label name="issue" value="http://10.1.101.66:890/index.php?m=testcase&amp;f=view&amp;caseID=45&amp;version=1"/>
                <label name="story" value="会员报表"/>
                <label name="feature" value="报表管理"/>
                <label name="host" value="DESKTOP-8DI1090"/>
                <label name="thread" value="15520@DESKTOP-8DI1090.main"/>
            </labels>
        </test-case>
        <test-case start="1576912736083" stop="1576912736085" status="canceled">
            <name>withdrawMoneyVip</name>
            <title>验证会员报表本月提现金额</title>
            <description type="text">判断会员报表的提现金额是否等于该会员提现记录的提现总计</description>
            <failure>
                <message>SkipException: The test was skipped for some reason</message>
                <stack-trace>org.testng.SkipException: The test was skipped for some reason
	at ru.yandex.qatools.allure.testng.AllureTestListener.onTestSkipped(AllureTestListener.java:148)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1678)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1670)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="severity" value="blocker"/>
                <label name="issue" value="http://10.1.101.66:890/index.php?m=testcase&amp;f=view&amp;caseID=45&amp;version=1"/>
                <label name="story" value="会员报表"/>
                <label name="feature" value="报表管理"/>
                <label name="host" value="DESKTOP-8DI1090"/>
                <label name="thread" value="15520@DESKTOP-8DI1090.main"/>
            </labels>
        </test-case>
        <test-case start="1576912736087" stop="1576912736090" status="canceled">
            <name>betNumber</name>
            <title>验证会员报表本月投注笔数</title>
            <description type="text">判断会员报表的投注笔数是否等于投注详情页的投注数量</description>
            <failure>
                <message>SkipException: The test was skipped for some reason</message>
                <stack-trace>org.testng.SkipException: The test was skipped for some reason
	at ru.yandex.qatools.allure.testng.AllureTestListener.onTestSkipped(AllureTestListener.java:148)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1678)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1670)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="severity" value="blocker"/>
                <label name="issue" value="http://10.1.101.66:890/index.php?m=testcase&amp;f=view&amp;caseID=45&amp;version=1"/>
                <label name="story" value="会员报表"/>
                <label name="feature" value="报表管理"/>
                <label name="host" value="DESKTOP-8DI1090"/>
                <label name="thread" value="15520@DESKTOP-8DI1090.main"/>
            </labels>
        </test-case>
        <test-case start="1576912736091" stop="1576912736094" status="canceled">
            <name>betMoney</name>
            <title>验证会员报表本月投注金额</title>
            <description type="text">判断会员报表的投注金额是否等于投注详情页的投注金额总计</description>
            <failure>
                <message>SkipException: The test was skipped for some reason</message>
                <stack-trace>org.testng.SkipException: The test was skipped for some reason
	at ru.yandex.qatools.allure.testng.AllureTestListener.onTestSkipped(AllureTestListener.java:148)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1678)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1670)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="severity" value="blocker"/>
                <label name="issue" value="http://10.1.101.66:890/index.php?m=testcase&amp;f=view&amp;caseID=45&amp;version=1"/>
                <label name="story" value="会员报表"/>
                <label name="feature" value="报表管理"/>
                <label name="host" value="DESKTOP-8DI1090"/>
                <label name="thread" value="15520@DESKTOP-8DI1090.main"/>
            </labels>
        </test-case>
        <test-case start="1576912736095" stop="1576912736097" status="canceled">
            <name>rechargeDiscount</name>
            <title>验证会员报表本月充值优惠金额</title>
            <description type="text">判断会员报表的充值优惠金额是否等于充值记录本月充值优惠金额总计</description>
            <failure>
                <message>SkipException: The test was skipped for some reason</message>
                <stack-trace>org.testng.SkipException: The test was skipped for some reason
	at ru.yandex.qatools.allure.testng.AllureTestListener.onTestSkipped(AllureTestListener.java:148)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1678)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1670)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="severity" value="blocker"/>
                <label name="issue" value="http://10.1.101.66:890/index.php?m=testcase&amp;f=view&amp;caseID=45&amp;version=1"/>
                <label name="story" value="会员报表"/>
                <label name="feature" value="报表管理"/>
                <label name="host" value="DESKTOP-8DI1090"/>
                <label name="thread" value="15520@DESKTOP-8DI1090.main"/>
            </labels>
        </test-case>
        <test-case start="1576912736100" stop="1576912736103" status="canceled">
            <name>agentMoney</name>
            <title>验证会员报表本月代理赔率金额</title>
            <description type="text">判断会员报表的代理赔率金额是否等于0</description>
            <failure>
                <message>SkipException: The test was skipped for some reason</message>
                <stack-trace>org.testng.SkipException: The test was skipped for some reason
	at ru.yandex.qatools.allure.testng.AllureTestListener.onTestSkipped(AllureTestListener.java:148)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1678)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1670)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="severity" value="blocker"/>
                <label name="issue" value="http://10.1.101.66:890/index.php?m=testcase&amp;f=view&amp;caseID=45&amp;version=1"/>
                <label name="story" value="会员报表"/>
                <label name="feature" value="报表管理"/>
                <label name="host" value="DESKTOP-8DI1090"/>
                <label name="thread" value="15520@DESKTOP-8DI1090.main"/>
            </labels>
        </test-case>
        <test-case start="1576912736104" stop="1576912736105" status="canceled">
            <name>agentWater</name>
            <title>验证会员报表本月代理退水金额</title>
            <description type="text">判断会员报表的代理退水金额是否等于0</description>
            <failure>
                <message>SkipException: The test was skipped for some reason</message>
                <stack-trace>org.testng.SkipException: The test was skipped for some reason
	at ru.yandex.qatools.allure.testng.AllureTestListener.onTestSkipped(AllureTestListener.java:148)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1678)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1670)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="severity" value="blocker"/>
                <label name="issue" value="http://10.1.101.66:890/index.php?m=testcase&amp;f=view&amp;caseID=45&amp;version=1"/>
                <label name="story" value="会员报表"/>
                <label name="feature" value="报表管理"/>
                <label name="host" value="DESKTOP-8DI1090"/>
                <label name="thread" value="15520@DESKTOP-8DI1090.main"/>
            </labels>
        </test-case>
        <test-case start="1576912736107" stop="1576912736108" status="canceled">
            <name>activityLoseWin</name>
            <title>验证会员报表本月实际输赢（不包括退水）金额</title>
            <description type="text">判断会员报表的实际输赢（不包括退水）金额是否等于投注详情的实际输赢金额减去退水金额</description>
            <failure>
                <message>SkipException: The test was skipped for some reason</message>
                <stack-trace>org.testng.SkipException: The test was skipped for some reason
	at ru.yandex.qatools.allure.testng.AllureTestListener.onTestSkipped(AllureTestListener.java:148)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1678)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1670)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="severity" value="blocker"/>
                <label name="issue" value="http://10.1.101.66:890/index.php?m=testcase&amp;f=view&amp;caseID=45&amp;version=1"/>
                <label name="story" value="会员报表"/>
                <label name="feature" value="报表管理"/>
                <label name="host" value="DESKTOP-8DI1090"/>
                <label name="thread" value="15520@DESKTOP-8DI1090.main"/>
            </labels>
        </test-case>
        <test-case start="1576912736110" stop="1576912736112" status="canceled">
            <name>activityLoseWinWater</name>
            <title>验证会员报表本月实际输赢（包括退水）</title>
            <description type="text">判断会员报表的实际输赢（包括退水）是否等于投注详情的输赢总金额</description>
            <failure>
                <message>SkipException: The test was skipped for some reason</message>
                <stack-trace>org.testng.SkipException: The test was skipped for some reason
	at ru.yandex.qatools.allure.testng.AllureTestListener.onTestSkipped(AllureTestListener.java:148)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1678)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1670)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="severity" value="blocker"/>
                <label name="issue" value="http://10.1.101.66:890/index.php?m=testcase&amp;f=view&amp;caseID=45&amp;version=1"/>
                <label name="story" value="会员报表"/>
                <label name="feature" value="报表管理"/>
                <label name="host" value="DESKTOP-8DI1090"/>
                <label name="thread" value="15520@DESKTOP-8DI1090.main"/>
            </labels>
        </test-case>
        <test-case start="1576912736113" stop="1576912736116" status="canceled">
            <name>activityWaterMoney</name>
            <title>验证会员报表本月实际退水金额</title>
            <description type="text">判断会员报表的实际退水金额是否等于投注金额*0.5%</description>
            <failure>
                <message>SkipException: The test was skipped for some reason</message>
                <stack-trace>org.testng.SkipException: The test was skipped for some reason
	at ru.yandex.qatools.allure.testng.AllureTestListener.onTestSkipped(AllureTestListener.java:148)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1678)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1670)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="severity" value="blocker"/>
                <label name="issue" value="http://10.1.101.66:890/index.php?m=testcase&amp;f=view&amp;caseID=45&amp;version=1"/>
                <label name="story" value="会员报表"/>
                <label name="feature" value="报表管理"/>
                <label name="host" value="DESKTOP-8DI1090"/>
                <label name="thread" value="15520@DESKTOP-8DI1090.main"/>
            </labels>
        </test-case>
    </test-cases>
    <labels>
        <label name="framework" value="TestNG"/>
        <label name="language" value="JAVA"/>
    </labels>
</ns2:test-suite>
